data/examples/declaration/value/function/operators-0-out.hs

```diff
@@ -1,5 +1,6 @@
 a =
-  b & c .~ d
+  b
+    & c .~ d
     & e
-      %~ f
```

Comment:
Here, it seems that the previous output accounted for operator precedence

Reply:
Yes, we should probably go back to having more indentation indicative of the actual precedence.

Reply:
Just in case, if at some point we decide to not display precedences anymore, see https://gist.github.com/tbagrel1/a5b9db62db23295113048ac410f30045

Reply:
Fixed with the last commit